beanprice/price.py

```diff
@@ -252,16 +262,16 @@ def log_currency_list(message, currencies):
         logging.debug("  {:>32}".format('{} /{}'.format(base, quote)))
 
 
-def get_price_jobs_at_date(entries,
-                           date=None,
-                           inactive=False,
-                           undeclared_source=None):
+def get_price_jobs_at_date(entries: List[data.Directive],
```

Comment:
Isn't `data.Entries` more appropriate here?

Reply:
Yes, changed to `data.Entries`.